### PR TITLE
edited README

### PR DESCRIPTION
**Pull request by shaur97**
lgtm

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # Cows and Bulls
+BCG ROX
```
Comment (rishabh-me) [inline README.md]: Could you change this so that it has the correct details?

**Review by shaur97 — COMMENTED** [review]
wow

File: README.md (hunk 1 above)
Comment (shaur97) [reply README.md]: braaah, its gots corrects details only maan 
but okay ill change it braaah

**Comment by shaur97** [comment]
closed

**Pull request closed without merging**